c10/core/TensorAxes.h

```diff
@@ -0,0 +1,204 @@
+#pragma once
+
+#include <c10/core/DefaultDtype.h>
+#include <c10/core/Device.h>
+#include <c10/core/Layout.h>
+#include <c10/core/ScalarType.h>
+#include <c10/core/TensorTypeSet.h>
+
+namespace c10 {
+
+/// `TensorAxes` holds the effectively used data type, device and layout of a
+/// `Tensor`. Additionally it indicates whether a `Tensor` is an autograd
+/// variable.
+///
+/// The class is in general immutable but allows to change the representation
```

Comment:
`but allows to change` doesn't work grammatically.  I think it should be something like `but the representation as a whole can be changed via ...`

Reply:
@resistor thanks a lot for the comment! As non native speaker the docs part is always 10x harder for me than writing pure code ;-). I highly appreciate any help/corrections on that side! :-)